src/styles/styles.js

```diff
@@ -1052,6 +1052,10 @@ const styles = {
         paddingVertical: 12,
     },
 
+    createPaymentMethodMenuContainer: {
+        top: 380,
+    },
```

Comment:
I don't see why do we need to pass the width now. If it was working fine on web without it.

Reply:
This is because `let position = getClickedElementLocation(nativeEvent);` has native implementation which returns just `{ bottom: 0, left: 0 }`

Reply:
Can't we use https://reactnative.dev/docs/0.64/pressevent to calculate the values?

Reply:
@parasharrajat Thanks for the docs. 
Looking at the example:
```
{
    changedTouches: [PressEvent],
    identifier: 1,
    locationX: 8,
    locationY: 4.5,
    pageX: 24,
    pageY: 49.5,
    target: 1127,
    timestamp: 85131876.58868201,
    touches: []
}
```
we could theoretically calculate the top left coordinate of the pressed component, by using `pageX`, `pageY` and `locationX`, `locationY`. We would still need to add a height value of the pressed component to `y` coordinate, to move the popover to its bottom.
What do you think about that?

Reply:
Ok. I think that's fine if height is needed let's calculate it.